docs/deploy.sh

```diff
@@ -4,11 +4,12 @@ git checkout -b docs-deploy-temp stable
 echo "Building _site"
 rake build
 echo "Staging new files"
-git add -A 
+git add -A
+git add --force -- _jekyll _site
```

Comment:
This could be non-obvious to unfamiliar readers. I would propose adding a comment:
```
git add --force -- _jekyll _site # Forcefully add the _jekyll and _site folder before pushing to Heroku, this is necessary because both folders are ignored under the main .gitignore
```